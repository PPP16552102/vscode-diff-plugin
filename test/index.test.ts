/// <reference types="vite/client" />

import { describe, expect, it } from 'vitest';

const fixtures = import.meta.glob('./fixtures/*.ts', { eager: true, as: 'raw' });

function transform(code: string) {
    return code.toLowerCase();
}

Object.entries(fixtures).forEach(([path, code]) => {
    it(path, () => {
        const result = transform(code);
        expect(result).toMatchSnapshot()
    })
})
