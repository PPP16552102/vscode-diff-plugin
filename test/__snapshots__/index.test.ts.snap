// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`./fixtures/index.ts 1`] = `
"CONSOLE.LOG('FOO');
"
`;
